wp-cli/vip-migrations.php

```diff
@@ -42,7 +42,7 @@ function cleanup( $args, $assoc_args ) {
 			$it = new \WP_CLI\Iterators\Table( $iterator_args );
 			foreach ( $it as $blog ) {
 				$url = $blog->domain . $blog->path;
-				$cmd = "--url={$url} vip migration cleanup";
+				$cmd = "--url={$url} vip migration cleanup --skip-confirm";
```

Comment:
Should we inherit from the main command like we do with dry-run? (Just in case).

Reply:
Cool, makes sense. 🚢